client/src/components/PetalModal.js

```diff
@@ -22,6 +23,7 @@ function PetalModal(props) {
     {
       petal: "Aspirations",
       questionOptions: [
+        "Please select a question below",
```

Comment:
Can we make the "next" button disabled if you have this option selected? That way we won't have any Flower petals with the question `"Please select a question below"`.